### PR TITLE
[INT-351] Billing notification preferences

### PR DESCRIPTION
**Pull request by cmunozgar**
### Description
Add new notification preference for Billing emails.

### Context & Notes
[INT-351] [Update buffer-publish to support new email notification group "Billing"](https://buffer.atlassian.net/browse/INT-351)

### Screenshots (if appropriate)
Screenshot 2021-07-05 at 15 06 05

### Best Practices Checklist

-   [X] My code follows the code style and guidelines of this project. 
-   [ ] I have added tests to cover my changes.
-   [X] I have tested different user stories.
-   [X] I have identified similar or related features and tested they are still working.
-   [X] I have performed a self-review of my own code.
-   [X] I have tested my changes/additions in the latest Chrome, Firefox, and Safari.
-   [X] I have commented my code, particularly in hard-to-understand areas.
-   [ ] I kept accessibility in mind by following the [a11y checklist.](https://www.notion.so/buffer/Workflow-Checklist-e64d86eb795140bcbfdc16d1c72e573f)
-   [ ] I have considered [security, abuse & compliance](https://www.notion.so/buffer/Engineering-Wiki-f34142d290304c35bebadf76cc9cc89e#cc6dcc7617184227b77da2e1b262a563) implications of my changes.

### ⚠️ Pre-deploy Checklist

- [ ] I have manually loaded the URL of the staging deployment and confirmed it's working (see below**)
- [ ] I have confirmed that the Cypress E2E test is passing (_Note: we have a known issue where it doesn't run on some PRs. Hopefully this will be fixed soon._)
- [ ] I have confirmed that all other tests are passing.

** To visit the URL of the staging deployment, please vlick the link in the bot comment, or look under checks at the bottom of this PR and click "Details" next to `bufferbotbrains/cicd-buffer-publish`
image

[INT-351]: https://buffer.atlassian.net/browse/INT-351

### CODE REVIEW

File: packages/i18n/translations/es-es.json [hunk 1]
```diff
@@ -367,7 +367,9 @@
       "movedBackToDrafts": "Aprobación pendiente a borrador",
       "movedBackToDraftsEmail": "Enviarme correos electrónicos cuando mi borrador se mueva de Aprobación pendiente a Borradores.",
       "celebrations": "Celebraciones",
-      "celebrationsEmail": "Enviarme correos electrónicos celebrando mis logros de publicación."
+      "celebrationsEmail": "Enviarme correos electrónicos celebrando mis logros de publicación.",
+      "billing": "Facturación",
+      "billingEmail": "Enviarme correos electrónicos sobre facturación y recordatorios de pago."
```
Comment (hamstu) [inline packages/i18n/translations/es-es.json:372]: Nice touch adding these! For context this was a great project by José and Maya, but they're not used, and we've decided to not invest in i18n right now as a team.

**Review by hamstu — APPROVED** [review]
LGTM! 👍🏻

**Review by MayaUribe — APPROVED** [review]
Nice one! Thank you @cmunozgar!! 🚀

File: packages/i18n/translations/es-es.json (hunk 1 above)
Comment (cmunozgar) [reply packages/i18n/translations/es-es.json:372]: It was easy for me to translate :)

**Pull request merged**